bin/restic_backup.sh

```diff
@@ -44,6 +44,10 @@ assert_envvars \
 IFS=':' read -ra backup_paths <<< "$RESTIC_BACKUP_PATHS"
 IFS=' ' read -ra extra_args <<< "$RESTIC_BACKUP_EXTRA_ARGS"
 
+#run the commands listed here before running the actual backup.
+while IFS= read -r command; do
```

Comment:
Hm what are the benefits of reading the commands as a text file with splitting, instead of having a shell script that is sourced or executed directly?

```bash
PRE_SCRIPT="$INSTALL_PREFIX/etc/restic/pre_backup.sh"
test -x "$PRE_SCRIPT" && $PRE_SCRIPT
# -or-
test -e "$PRE_SCRIPT" && . $PRE_SCRIPT
```

Reply:
actually none. :)